ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -874,23 +874,23 @@ const triggerSet: TriggerSet<Data> = {
         const defamation = data.defamationColor;
         if (defamation === undefined)
           return;
-        if (regression === 'remote') {
-          const color = defamation === 'red' ? output['blue']!() : output['red']!();
-          return output.nearTether!({ color: color });
-        }
+
+        const defamationTowerColor = defamation === 'red' ? output.red!() : output.blue!();
```

Comment:
Also just cleaning this up to call `output.thing!()` directly, since there's better testing for that, as `output[thing]!()` is runtime checking only.